mod/05_standard/set/type/uri.rb

```diff
@@ -0,0 +1,11 @@
+format do
+  include Phrase::Format
+
+  view :core do |args|
+    build_link _render_raw(args), render_title(args) || card.name
+  end
+
+  view :uri do |args|
+    build_link _render_raw(args)
```

Comment:
OK, here's what I would propose:
1. As implemented, we use URI for the cardtype name/codename
2. In views, we make uri and url synonymous.  I know that technically they're not, but for now the effort to make it so that "url" balks at non-url uri's seems a bit unnecessary.
3. Both of those views should actually return simple uris in pretty much every format. In other words, something like `http://wagn.org`, NOT `<a href="http://wagn.org">http://wagn.org</a>`
4. For that latter case, we can create a new view called "uri_link".  In many formats this will just be a simple uri, but in HtmlFormat it will have a hypertext link.
5. For all of the following views: uri, url, and uri_link, MOST cards will use the url of the card itself (based on the name), but URI cards will use the content url.

How does that sound?